src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -119,12 +119,14 @@ def _encode_name(backend, attributes):
     for attribute in attributes:
         value = attribute.value.encode('utf8')
         obj = _txt2obj_gc(backend, attribute.oid.dotted_string)
+        if attribute.oid == NameOID.COUNTRY_NAME:
+            # Per RFC5280 countryName should be encoded as PrintableString,
+            # not UTF8String
```

Comment:
Sure. It's Appendix A.1: `X520countryName ::=     PrintableString (SIZE (2))`. Supposedly X.520 contains the normative description as well.